### DISCUSSION TITLE
Clear effects in radius by Guid

### DISCUSSION THREAD

**Discussion by IlyaSergeevich0** [SmartlyDressedGames/Unturned-3.x-Community#4094] [Ideas]
I did not find such an option in `EffectManager` :(

Basically, I have a multiple effects in different points. But in some cases I need to remove only exact ones.

`EffectManager.ClearEffectByGuid` removes all of them.

Would be cool to see a feature, that clear them in some radius since `EffectManager.pool` stores GameObject's.

**Comment by DanielWillett** [comment]
+1, currently you have to keep all your effects managed, wipe them all, then replace the ones you want. Quite a pain
